tests/tests.sh

```diff
@@ -10,6 +10,7 @@ if [ ! -f "env/bin/activate" ]
 fi
 
 source env/bin/activate
+pip install -q -U -e ../python/
```

Comment:
Dev question: If I've done `source env/bin/activate` manually and I'm running `ctest ...` rather than `./tests.sh ...`, do I need to rerun this command each time I make changes under `python/ccf`? Or are changes picked up automagically since its a local project?

Reply:
Just tested it to confirm: changes get picked up automagically and there's no other command to run.